core/arxiv/submission/services/classic/models.py

```diff
@@ -320,6 +317,9 @@ def update_from_submission(self, submission: domain.Submission) -> None:
                 and self.status in [Submission.NOT_SUBMITTED, None]:
             self.status = Submission.SUBMITTED
             self.submit_time = submission.updated
+        # Delete.
+        elif submission.deleted:
+            self.status = Submission.DELETED_USER
```

Comment:
Unsure where the _USER part comes from - is this a required state from the classic system? maybe a comment to clarify, since everywhere else the suffix isn't used?

Reply:
It's one of the states for the status column in the classic database. See https://github.com/cul-it/arxiv-submission-core/blob/feature/ARXIVNG-904/core/arxiv/submission/services/classic/models.py#L61

Reply:
Oh, I see. Got it.